### PR TITLE
NMSW-35 Create a cookie policy page

### PR DESCRIPTION
**Pull request by Adele-B**
# Ticket

NMSW-35

----

## AC

----

## To test

- run app locally
- go to localhost:3000
- In the footer
- > You should see a link that says cookies
- click on the cookie link
- > You should be taken to /cookies
- > You should see a heading with cookies

----

### CODE REVIEW

File: src/pages/Footer/__tests__/CookiePolicy.test.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import CookiePolicy from '../CookiePolicy';
+
+describe('Cookie policy tests', () => {
+
+  it('should render a title of Cookies', async () => {
+    render(<CookiePolicy />);
+    expect(screen.getByText('Cookies')).toBeInTheDocument();
```
Comment (jennikate) [inline src/pages/Footer/__tests__/CookiePolicy.test.jsx:8]: We could add a second assertion that the Cookie link has the correct href in the html - similar to
`expect(checkCrownCopyrightLogo.outerHTML).toEqual('<a class="govuk-footer__link govuk-footer__copyright-logo" href="/">© Crown copyright</a>');`

File: src/layout/__tests__/Footer.test.jsx [hunk 2]
```diff
@@ -1,15 +1,16 @@
...
+    await waitFor(() => { render(<MemoryRouter><Footer /></MemoryRouter>); });
     expect(screen.getByText('© Crown copyright')).toBeInTheDocument();
   });
 
   it('should render the Crown Copyright element with the class that contains the image', async () => {
-    await waitFor(() => { render(<Footer />); });
+    await waitFor(() => { render(<MemoryRouter><Footer /></MemoryRouter>); });
     const checkCrownCopyrightLogo = screen.getByText('© Crown copyright');
     expect(checkCrownCopyrightLogo).toBeInTheDocument();
     expect(checkCrownCopyrightLogo.outerHTML).toEqual('<a class="govuk-footer__link govuk-footer__copyright-logo" href="/">© Crown copyright</a>');
```
Comment (jennikate) [inline src/layout/__tests__/Footer.test.jsx:16]: Might be worth adding a test to check the Cookie link loads in with the footer component

**Review by jennikate — APPROVED** [review]
Added a few suggestions - all just suggestions! So see what you think :)

Overall LGTM

File: src/pages/Footer/CookiePolicy.jsx [hunk 3]
```diff
@@ -0,0 +1,9 @@
+const CookiePolicy = () => {
```
[the remark this answers is not in the document]
Comment (Adele-B) [reply src/pages/Footer/CookiePolicy.jsx:1]: Will change folder name on last regulatory information page PR

**Pull request merged**